### ISSUE TITLE
Can't easily update existing file

### ISSUE THREAD

**Issue by siggipop** [fsxmanu/dynamics-for-dev#27]
Can't figure out why, on upload, there is always a new file created instead of updating!? 
if I select prefix  xyz_ it just creates a new fil and appends prefix, so end result is xyz_xyz_[filename]
if i omit prefix then a new file is created with name new_xyz_[filename] 
i just want to updated the same file that I downloaded prior: xyz_[filename]

**Comment by fsxmanu** [comment]
Hi

Can you show me your dynamicsConfig.json file? Of course without your dynamics URL. And the filename you are uploading is "xyz_[filename]" correct?
How do you upload it to dynamics? Do you right click the file and click the upload option?

**Comment by siggipop** [comment]
(I changed some values, to mask any identifiers)
Filename: "xyz_contact_script_library.js"

When uploading, I do:
Right-click file -> "Upload Webresource to Dynamics" -> select folder "/" -> select prefix "xyz_" 
Now here the message I get, is: "file does not exist. Create new?"

```
{
    "NamingConvention" : {
        "WebResourceFolder" : [ "/", "/Contact"],
        "Prefix" : [ "", "xyz_" ]
    },
    "OrgInfo" : {
        "CrmUrl" : "https://companyname.crm4.dynamics.com/",
        "ApiVersion" : "9.2"
    },
    "UploadOptions" : {
        "AddExistingToSolution" : false
    },
    "Solutions" : ["solutionname" ]
}
```

**Comment by fsxmanu** [comment]
Thx I will test it now on my end. I could add a condition that if the selected prefix is already in the filename it just uses that. I will report back on this

**Comment by fsxmanu** [comment]
I'm back. I just released version 0.7.6 which should resolve this issue. Could you try it with the new version?

**Comment by siggipop** [comment]
Oh ok. That explaines a bit. I made it work by just deleting the publisher prefix part of the file locally (after download), and used the same prefix when uploading. Now that works. 
I noticed another issue. The button "Upload Webresource to Dynamics" is sometimes missing when file does not have extension: ".js"
But when downloading a js webresource dynamics environment, the logical name _can_ contain .js extension or not! depends how it was named in PowerPlatform editor.

**Comment by fsxmanu** [comment]
Oh ok so if it is a js file the name does not have a .js extension? Which ending does that file then have? just none?
I try to filter it to file extensions because I don't want that button to show everytime someone right clicks a file.

**Comment by siggipop** [comment]
> I'm back. I just released version 0.7.6 which should resolve this issue. Could you try it with the new version?
> Also use the "empty" prefix you had in your options. I will add the matching prefix options in a future release but this one is so you can try it with that.

Yeah, I will try to acquire latest version and test later.

**Comment by siggipop** [comment]
> Oh ok so if it is a js file the name does not have a .js extension? Which ending does that file then have? just none? I try to filter it to file extensions because I don't want that button to show everytime someone right clicks a file.

Yeah that's correct, the file can be of uknown type, without any extension! MS Dynamics allows that!? probably because content  is already wrapped inside webresource entity with type: JScript, so there is no confusion.

**Comment by fsxmanu** [comment]
> 
> Yeah that's correct, the file can be of uknown type, without any extension! MS Dynamics allows that!? probably because content is already wrapped inside webresource entity with type: JScript, so there is no confusion.

Well that's nice. I will see if I can enable it for unknown file types without enabling it globally

**Comment by fsxmanu** [comment]
> > I'm back. I just released version 0.7.6 which should resolve this issue. Could you try it with the new version?
> > Also use the "empty" prefix you had in your options. I will add the matching prefix options in a future release but this one is so you can try it with that.
> 
> Yeah, I will try to acquire latest version and test later.
> 
> edit: ...or I could just test right away.. I tried uploading with no-prefix option and while the file contained original prefix, and it worked beautifully..👍

Good to hear! I will look into that unknown file type thing tomorrow. 
Idk if you have seen it but I added a new experimental pathresolver setting for the json. if you want you can try it. It is however experimental and I need to find a better solution for it

**Comment by fsxmanu** [comment]
@siggipop 
So there should be a new version which should allow you to upload a file without a extension. Took a bit longer sorry for that
